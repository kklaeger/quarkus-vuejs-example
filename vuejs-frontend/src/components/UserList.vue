<template>
  <div>
    <h1>{{ title }}</h1>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator";
@Component({
  components: {}
})
export default class UserList extends Vue {
  private title = "User List";
}
</script>

<style scoped lang="scss"></style>
