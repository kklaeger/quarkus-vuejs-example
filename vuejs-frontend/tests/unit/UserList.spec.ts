import { shallowMount } from "@vue/test-utils";
import UserList from "@/components/UserList.vue";

describe(UserList.name + " component", () => {
  it("should render successfully.", () => {
    const wrapper = shallowMount(UserList);
    expect(wrapper).toMatchSnapshot();
  });
});
