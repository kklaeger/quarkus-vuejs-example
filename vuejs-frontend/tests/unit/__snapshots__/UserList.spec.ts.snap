// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserList component should render successfully. 1`] = `
<div>
  <h1>User List</h1>
</div>
`;
